ts/src/mexc.ts

```diff
@@ -121,6 +121,9 @@ export default class mexc extends Exchange {
                         'public': 'https://contract.mexc.com/api/v1/contract',
                         'private': 'https://contract.mexc.com/api/v1/private',
                     },
+                    'broker': {
```

Comment:
@sc0Vu there's no use in merging a broken API, if you cannot figure out how to do the signature we can try to reach out to them

Reply:
Actually the error is right we're not sending `signature` in the request

```
mexc.brokerPrivateGetSubAccountList ()
fetch Request:
 mexc GET https://api.mexc.com/api/v3/broker/sub-account/list 
RequestHeaders:
 {
  ApiKey: 'mx0vgl6vNvbxWo2Nwj',
  'Request-Time': '1685532706554',
  'Content-Type': 'application/json',
  source: 'CCXT',
  Signature: 'ff8d9f9b81bea1c09800790ffdeebec6e6557ea0265efe407ca66af656fcdad2'
} 
RequestBody:
 undefined 

handleRestResponse:
 mexc GET https://api.mexc.com/api/v3/broker/sub-account/list 400 Bad Request 
ResponseHeaders:
 {
  'Akamai-Grn': '0.af6e5668.1685532706.277ff1d5',
  Connection: 'keep-alive',
  'Content-Length': '99',
  'Content-Type': 'application/json',
  Date: 'Wed, 31 May 2023 11:31:46 GMT',
  'Server-Timing': 'cdn-cache; desc=MISS, edge; dur=160, origin; dur=72, ak_p; desc="468203_1750494895_662696405_23274_4523_0_-";dur=1'
} 
ResponseBody:
 {"code":700004,"msg":"Mandatory parameter 'signature' was not sent, was empty/null, or malformed."} 
```